driver-mapping/src/test/java/com/datastax/driver/mapping/MapperSaveNullFieldsTest.java

```diff
@@ -40,6 +41,7 @@ public void setup() {
         mapper = new MappingManager(session()).mapper(User.class);
     }
 
+    @CassandraVersion("2.2.0")
```

Comment:
Should be 2.1.0 right?  The problem only happens on 1.2/2.0 because client timestamps aren't used (happens in event when both inserts are done in same millisecond)